helm/net-exporter/templates/daemonset.yaml

```diff
@@ -55,6 +55,17 @@ spec:
         args:
         - "-namespace={{ .Release.Namespace }}"
         - "-timeout={{ .Values.timeout }}"
+        {{ if (.Values.Installation) }}
+        {{ if (.Values.Installation.V1.Monitoring.NetExporter.Hosts) }}
+        - "-hosts={{ .Values.Installation.V1.Monitoring.NetExporter.Hosts }}"
+        {{ end }}
+        {{ if (.Values.Installation.V1.Monitoring.NetExporter.NTPServers) }}
+        - "-ntp-servers={{ .Values.Installation.V1.Monitoring.NetExporter.NTPServers }}"
+        {{ end }}
+        {{ if (.Values.Installation.V1.Monitoring.NetExporter.DNSCheck.TCP.Disabled) }}
+        - "-disable-dns-tcp-check={{ .Values.Installation.V1.Monitoring.NetExporter.DNSCheck.TCP.Disabled }}"
```

Comment:
For my information, why is this configurable both from `.Values.Installation` and `.Values.NetExporter`?

Reply:
Nevermind, just saw https://github.com/giantswarm/installations/pull/1322#issuecomment-651579222